DESCRIPTION

```diff
@@ -38,5 +38,13 @@ Collate:
     'statistic-tree.R'
     'statistic-variance.R'
     'update_parameters.R'
+    'utilities-categorical.R'
+    'utilities-covariance.R'
+    'utilities-data-validation.R'
+    'utilities-histogram.R'
+    'utilities-noise-generation.R'
+    'utilities-objective.R'
+    'utilities-postprocessing.R'
```

Comment:
I first thought that `R/createJSON.R` should be moved into the postprocessing utilities file, but it seems to be dead/rotten:

- only the histogram and mean implement a .getJSON
- we have vignettes now, is this system still necessary?
- if they are necessary, it would be better to move them into the class to avoid a sketchy `match.fun` call

Reply:
I agree about moving the function. Discussion needs to continue about whether or not the function should stay in the library.